src/Sulu/Component/HttpKernel/SuluKernel.php

```diff
@@ -149,9 +151,6 @@ protected function load(LoaderInterface $loader, $confDir, $pattern)
         }
     }
 
-    /**
-     * @param RouteCollectionBuilder|RoutingConfigurator $routes Is a RouteCollectionBuilder for Symfony <= 4.4
-     */
     protected function import($routes, $confDir, $pattern)
```

Comment:
```suggestion
    /**
     * * @param RouteCollectionBuilder|RoutingConfigurator $routes Is a RouteCollectionBuilder for Symfony <= 4.4
     */
    protected function import($routes, $confDir, $pattern)
```

phpstan requires this.